Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP.py

```diff
@@ -0,0 +1,3528 @@
+        offset_multiplier = 1
+
+    # No Pagination
+    else:
+        number_of_requests = 1
+        limit = MAX_PAGE_SIZE
+        offset = None
+        offset_multiplier = 1
+
+    return Pagination(page, page_size, limit, offset, number_of_requests, offset_multiplier, is_automatic, is_manual)
+
+
+def delete_keys_from_dict(dictionary: MutableMapping, keys_to_delete: List[str] | Set[str]) -> Dict[str, Any]:
+    """
+    Get a modified dictionary without the requested keys
+
+    Args:
+        dictionary (Dict[str, Any]): Dictionary to modify according to.
+        keys_to_delete (List[str]): Keys to not include in the modified dictionary.
+
+    Returns:
+        Dict[str, Any]: Modified dictionary without requested keys.
+    """
+    keys_set = set(keys_to_delete)
+    modified_dict: Dict[str, Any] = {}
+
+    for key, value in dictionary.items():
+        if key not in keys_set:
+            if isinstance(value, MutableMapping):
+                modified_dict[key] = delete_keys_from_dict(value, keys_set)
+
+            elif isinstance(value, MutableSequence) \
+                    and len(value) > 0 \
+                    and isinstance(value[0], MutableMapping):
+                modified_dict[key] = []
+
+                for val in value:
+                    modified_dict[key].append(delete_keys_from_dict(val, keys_set))
+
+            else:
+                modified_dict[key] = copy.deepcopy(value)
+
+    return modified_dict
+
+
+def add_item_to_all_dictionaries(dictionaries: List[Dict[str, Any]], key: str, value: Any) -> None:
+    for dictionary in dictionaries:
+        dictionary[key] = value
+
+
+def is_query_correct(query: str = None) -> bool:
+    """
+    Check if the query format is correct
+
+    Args:
+        query (str, optional): Query string in some format.
+            Defaults to None.
+
+    Returns:
+        bool: Whether the format is correct or not
+    """
+    if not query:
+        return False
+
+    return not sha256Regex.match(query) \
+        and not re.match(ipv4Regex, query) \
+        and not re.match(urlRegex, query)
+
+
+def get_dbotscore(sha256: str = None, disposition: str = None) -> Common.DBotScore:
+    """
+    Get XSOAR score for the file's disposition.
+
+    Args:
+        sha256 (str, optional): SHA256 of the file.
+            Defaults to None.
+        disposition (str, optional): 3rd party score of the file's disposition.
+            Defaults to None.
+
+    Returns:
+        Common.DBotScore: DBot Score according to the disposition.
+    """
+    if disposition == 'Malicious':
+        score = Common.DBotScore.BAD
+
+    elif disposition == 'Clean':
+        score = Common.DBotScore.GOOD
+
+    else:
+        score = Common.DBotScore.NONE
+
+    return Common.DBotScore(
+        indicator=sha256,
+        indicator_type=DBotScoreType.FILE,
+        integration_name='CiscoAMP',
+        malicious_description=disposition,
+        reliability=DBotScoreReliability.C,
```

Comment:
do we not have an integration parameter for this value?

Reply:
Thanks for enlightening me about this parameter. 
I've added it and its functionality.